tools/topology/topology2/cavs-smart-amp-nocodec.conf

```diff
@@ -0,0 +1,235 @@
+<searchdir:include>
+<searchdir:include/common>
+<searchdir:include/components>
+<searchdir:include/dais>
+<searchdir:include/pipelines/cavs>
+<searchdir:platform/intel>
+
+<vendor-token.conf>
+<manifest.conf>
+<pdm_config.conf>
+<tokens.conf>
+<sink_pin_binding.conf>
+<src_pin_binding.conf>
+<virtual.conf>
+<smart-amp-playback.conf>
+<passthrough-playback.conf>
+<passthrough-capture.conf>
+<passthrough-be.conf>
+<data.conf>
+<pcm.conf>
+<pcm_caps.conf>
+<fe_dai.conf>
+<ssp.conf>
+<dmic.conf>
+<hw_config.conf>
+<manifest.conf>
+<route.conf>
+<common_definitions.conf>
+<copier.conf>
+<pipeline.conf>
+<dai.conf>
+<host.conf>
+<smart_amp.conf>
+<dmic-default.conf>
+
+Define {
+    MCLK    24576000
+}
+
+Object.Dai {
+	SSP."0" {
+		id 		0
+		dai_index	0
+		direction	"duplex"
+		name		NoCodec-0
+		default_hw_conf_id	0
+		sample_bits		32
+		quirks			"lbm_mode"
+		io_clk			$MCLK
+
+		Object.Base.hw_config."SSP0" {
+			id		0
+			mclk_freq	$MCLK
+			bclk_freq	6144000
+			format		DSP_B
+			tdm_slot_width	32
+			tdm_slots	4
+			rx_slots	15
+			tx_slots	15
+		}
+	}
+}
+
+Object.Pipeline {
+	# smart amp playback pipeline
+	smart-amp-playback.1 {
+		index	1
+		Object.Widget.pipeline.1 {
+			stream_name 'NoCodec-0'
+		}
+		Object.Widget.copier.1 {
+			stream_name 'SSP0 Playback'
+		}
+
+		Object.Widget.smart_amp.1 {
+			Object.Base.sink_pin_binding.1 {
+				sink_pin_binding_wname	"copier.host.1.1"
+			}
+
+			Object.Base.sink_pin_binding.2 {
+				sink_pin_binding_wname	"copier.SSP.4.1"
+			}
+
+			Object.Control.bytes."1" {
+				name smart_amp_init
+				access  [
+					tlv_read
+					tlv_callback
+				]
+				Object.Base.extops."extctl" {
+					get 258
+					put 0
+				}
+				max 4096
+				Object.Base.data."smart_amp_init" {
+					bytes "
+						0x53, 0x4f, 0x46, 0x34, 0x84, 0x00, 0x00, 0x00,
+						0x00, 0x00, 0x00, 0x04, 0x00, 0x00, 0x00, 0x00,
+						0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+						0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+
+						0x02, 0x00, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00,
+						0x00, 0x00, 0x00, 0x00, 0x14, 0x00, 0x00, 0x00,
+
+						0x00, 0x00, 0x00, 0x00, 0x80, 0x01, 0x00, 0x00,
+						0x80, 0xbb, 0x00, 0x00, 0x20, 0x00, 0x00, 0x00,
+						0x10, 0xff, 0xff, 0xff, 0x01, 0x00, 0x00, 0x00,
+						0x00, 0x00, 0x00, 0x00, 0x02, 0x20, 0x00, 0x00,
+
+						0x01, 0x00, 0x00, 0x00, 0x00, 0x03, 0x00, 0x00,
+						0x80, 0xbb, 0x00, 0x00, 0x20, 0x00, 0x00, 0x00,
+						0x10, 0xff, 0xff, 0xff, 0x01, 0x00, 0x00, 0x00,
+						0x00, 0x00, 0x00, 0x00, 0x04, 0x20, 0x00, 0x00,
+
+						0x00, 0x00, 0x00, 0x00, 0x00, 0x03, 0x00, 0x00,
+						0x80, 0xbb, 0x00, 0x00, 0x20, 0x00, 0x00, 0x00,
+						0x10, 0xff, 0xff, 0xff, 0x01, 0x00, 0x00, 0x00,
+						0x00, 0x00, 0x00, 0x00, 0x04, 0x20, 0x00, 0x00,
+
+						0x04, 0x00, 0x00, 0x00, 0x00, 0x01, 0xff, 0xff,
+						0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0x00, 0x01,
+						0xff, 0xff, 0xff, 0xff"
+				}
+			}
+		}
+	}
+
+	passthrough-be.1 {
```

Comment:
I don't want to see ANY references to passthrough moving forward. I have repeated over and over that we do not do passthrough pipelines any longer. We want mixers on playback and demux on capture. Get on with the program please.